Alignment/CommonAlignmentMonitor/plugins/AlignmentMonitorAsAnalyzer.cc

```diff
@@ -140,8 +141,10 @@ void AlignmentMonitorAsAnalyzer::analyze(const edm::Event& iEvent, const edm::Ev
     iSetup.get<IdealGeometryRecord>().get(theGeometricDet);
     edm::ESHandle<PTrackerParameters> ptp;
     iSetup.get<PTrackerParametersRcd>().get(ptp);
+    edm::ESHandle<PTrackerAdditionalParametersPerDet> ptitp;
```

Comment:
@adewit now that PR https://github.com/cms-sw/cmssw/pull/34270 is merged, this file has conflicts.
Can you please resolve them, and while you are at it use `esConsumes` for the newly introduced record?
Thanks.

Reply:
Done